File: Week 1 C/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get user input for blocks height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    for (int i = 1; i <= height; i++)
    {
        // Print spaces
        for (int j = 0; j < height - i; j++)
        {
            printf(" ");
        }
        // Print left blocks
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        // Print spaces
        printf("  ");
        // Print right blocks
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
    return 0;
}

File: Week 6 Python/sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    # Get user input for blocks height
    height = 0
    while height < 1 or height > 8:
        height = get_int("Height: ")

    for i in range(1, height + 1):
        # Print spaces
        for j in range(height - i):
            print(" ", end="")

        # Print left blocks
        for j in range(i):
            print("#", end="")

        # Print spaces
        print("  ", end="")

        # Print right blocks
        for j in range(i):
            print("#", end="")

        print()


if __name__ == "__main__":
    main()
